experimental/traffic-portal/src/app/core/ds-card/ds-card.component.html

```diff
@@ -57,7 +57,7 @@ <h3 *ngIf="deliveryService.exampleURLs">URLs</h3>
 		</div>
 	</mat-card-content>
 	<mat-card-actions *ngIf="open">
-		<a mat-raised-button color="accent" class="view-details" routerLink="/core/deliveryservice/{{deliveryService.id}}" title="View Details" aria-label="View Details">
+		<a id="{{deliveryService.xmlId}}Details" mat-raised-button color="accent" class="view-details" routerLink="/core/deliveryservice/{{deliveryService.id}}" title="View Details" aria-label="View Details">
```

Comment:
I have a similar complaint here; the equivalent selector being `mat-card#XMLID mat-card-actions > a`.